ApplicationConfigProviders/Aggregator.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformUIBundle\ApplicationConfigProviders;
+
+/**
+ * Aggregates configuration from ApplicationConfigProviders
+ */
+class Aggregator
+{
+    /** @var ApplicationConfigProvider[] */
+    private $providers = [];
+
+    public function addProviders( array $providers )
```

Comment:
this method deserves an API doc explaining that `$providers` is supposed to be a hash and that the keys are used in the config returned by `getConfig`